### ISSUE TITLE
Another 'System is not defined' issue

### ISSUE THREAD

**Issue by lukaskostial** [systemjs/builder#673]
Hi,

I've just updated jspm from **0.16.33** to **0.16.44** and I'm having the issue with SFX build:

- **System is not defined**

which is due to **System._nodeRequire** in **nodelibs-process@0.1.2** not converted to **$__System._nodeRequire**:

```javascript
$__System.registerDynamic('2e6', ['2e5'], true, function ($__require, exports, module) {
  var define,
      global = this || self,
      GLOBAL = global;
  module.exports = System._nodeRequire ? process : $__require('2e5');
  return module.exports;
});
```

This was definitely working with the older systemjs-builder. Is this regression or am I doing something wrong? 

Thank you.

**Comment by jrauschenbusch** [comment]
Sounds to me like there's a problem related to the cjs transformer upgrade (released with systemjs-builder v0.15.27). The necessary internal mapping of ```System``` to the global ```$__System``` is not performed during the transformation phase. The problem should occur starting with jspm v0.16.43.

There is a option parameter (opts.systemGlobal) which should be used by the cjs transformer to accomplish the mapping. It seems the problem is located here:
https://github.com/systemjs/babel-plugin-transform-cjs-system-wrapper/blob/master/src/index.js#L54

The following issues are replicas of this ticket:
https://github.com/jspm/jspm-cli/issues/2039
#674

**Comment by JockVanDallas** [comment]
i reverted jspm from 0.16.44 to 0.16.42 and it works now. Hope this will be fixed in future version, so i can use the latest jspm again.

**Comment by jrauschenbusch** [comment]
I've fixed the issue in the transformer and added a dedicated unit test.

@guybedford Finally we need a new release of the [babel-plugin-transform-cjs-system-wrapper](https://github.com/systemjs/babel-plugin-transform-cjs-system-wrapper) (v0.2.0?) to npm and a dependency version update in the package.json of the systemjs-builder comprising a new release ` systemjs-builder 0.15.30`/`jspm-cli 0.16.45`.

**Comment by guybedford** [comment]
@jrauschenbusch thanks so much for the quick update here. I've posted out a patch release and updated in https://github.com/systemjs/builder/commit/62e48110176a18892d65e97b9bd6a29fea2edc8b. Just waiting for CI tests to pass for the patch updates through SystemJS builder and jspm.

**Comment by guybedford** [comment]
Released in 0.15.30.
